net/ProfileViewController.m

```diff
@@ -67,7 +68,8 @@ - (void)viewDidAppear:(BOOL)animated {
     self.activityIndicatorView.hidden = NO;
     [self.activityIndicatorView startAnimating];
     
-    if ([[self backViewController] isKindOfClass:[IssueHistoryViewController class]]) {
+    if ([[self backViewController] isKindOfClass:[IssueHistoryViewController class]] ||
```

Comment:
This is Evgenij part of code (profile controller). I had to add my part for profile properly working